docs/faq.md

```diff
@@ -161,13 +160,43 @@ See [the Semgrep development philosophy](/contributing/semgrep-philosophy/) for
+
+#### SCA
+
+Snyk offers reachability analysis for Java, JavaScript, and TypeScript, while Semgrep offers reachability analysis for [multiple languages, including Java, JavaScript, and Ruby](supported-languages/#semgrep-supply-chain)
+
+Snyk offers limited information on whether dependencies are direct or transitive. However, this information is only available with Enterprise plans, and the information is limited to projects using Node.js, specifically npm and Yarn packages, and Maven. Semgrep Supply Chain offers advanced reachability analysis for direct dependencies in the form of dataflow reachability. Semgrep offers this coverage for seven languages and counting.
+
+Semgrep and Snyk both offer license compliance features, ensuring that the dependencies that your developers use meet the requirements set by your organization.
+
+To help you manage your findings, Semgrep provides you with the findings' EPSS probabilities, severity levels, transitivity information, and multiple levels of dataflow reachability. Snyk assesses impact and likelihood and encapsulates this information into a risk score. 
+
+#### Policies and rules management 
+
+Semgrep Code and Semgrep Secret's policies management feature provides extensive flexibility, especially with respect to a developer's workflow, by allowing results to appear:
+
+- Only in the AppSec team’s view (monitor mode)
+- In the AppSec team's view **and** in the developer’s workflow, while not failing the CI job (comment mode)
+- In the AppSec team's view **and** in the developer’s workflow, while also failing the CI job (block mode)
+
+Semgrep Supply Chain results in a failed CI job only when there are critical or high-severity findings.
 
-For SAST, both Semgrep and Snyk offer good solutions out of the box, however writing custom rules is easier and more scalable with Semgrep, enabling you to accommodate issues specific to your codebase that no vendor could out-of-box. Semgrep supports 30+ languages whereas Snyk supports 14.
+#### Secrets detection
 
-For secrets scanning, Semgrep Secrets leverages semantic analysis, entropy analysis, and validation to accurately detect and fix secrets. Snyk has a [business partnership with GitGuardian](https://blog.gitguardian.com/were-teaming-up-with-snyk-to-strengthen-developer-security/) to offer their secret scanning to Snyk customers.
+Semgrep Secrets leverages semantic analysis, entropy analysis, and validation to accurately detect and fix secrets. Snyk maintains a [business partnership with GitGuardian](https://blog.gitguardian.com/were-teaming-up-with-snyk-to-strengthen-developer-security/) to offer secrets scanning as part of Snyk Code.
 
-See the [Semgrep vs. Snyk webpage](https://semgrep.dev/resources/semgrep-vs-snyk) for a more detailed comparison between the two.
+> See [Semgrep vs. Snyk](https://semgrep.dev/resources/semgrep-vs-snyk) for a detailed comparison.
```

Comment:
```suggestion
```
Not sure if this is up-to-date, I am aware that the team is working on a new Snyk page, would suggest removing the above line and then add in the new page when it's available.